src/configuration/useAuthorityLinks.ts

```diff
@@ -0,0 +1,39 @@
+import links from "./links.json"
+
+const DEFAULT_LOCALE = "en"
+type TranslationValue = Record<string, string>
+type LinkData = {
+  url: string
+  label: TranslationValue
+}
+type LinkKeys = "about" | "legal"
+type AuthorityLinksContent = Record<LinkKeys, LinkData[]>
+type DisplayableLink = {
+  url: string
+  label: string
+}
+
+const useAuthorityLinks = (
+  key: LinkKeys,
+  localeCode: string,
+): DisplayableLink[] => {
+  try {
+    const authorityLinks = links as AuthorityLinksContent
+    const linksForKey = authorityLinks[key] || []
+
+    const linksOverride = linksForKey
+      .filter(({ label }) => {
+        const localizedLabel = label[localeCode] || label[DEFAULT_LOCALE]
+        return localizedLabel.length > 0
```

Comment:
Do we need to be filtering for this? we have ultimate control over the data, so we could just not include any entries that dont have labels.

Reply:
We do have control over the data, but there is zero validation on the input. It's just a file that gets downloaded. Not sure where can we gain more control and shift the complexity away from here. Still seems like an external dependency that we don't control the constraints off. I thought that this could make the part out of our control less disruptive of the functionality. But I also see how it is introducing complexity to it.